### PR TITLE
feat: add dashboard for visualizing currency exchange rates

### PR DESCRIPTION
**Pull request by wojciechka**

### CODE REVIEW

File: currency_exchange_rates/currency_exchange_rates.yml [hunk 1]
```diff
@@ -0,0 +1,363 @@
...
+kind: Label
+metadata:
+    name: Exchange Rates
+spec:
+    color: '#7A65F2'
+---
+apiVersion: influxdata.com/v2alpha1
+kind: Label
+metadata:
+    name: outputs.influxdb_v2
```
Comment (jsteenb2) [inline currency_exchange_rates/currency_exchange_rates.yml]: this name doesn't feel very descriptive 🤔

File: currency_exchange_rates/README.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  - 1 Variables: `bucket`
+
+## Setup Instructions
+    
+  The data for the dashboard is populated by the included Telegraf configuration. The Telegraf Configuration requires the following environment variables
+    
+  - `INFLUX_TOKEN` - The token with the permissions to read Telegraf configs and write data to the `telegraf` bucket. You can just use your master token to get started.
+  - `INFLUX_ORG` - The name of your Organization
+  - `INFLUX_HOST` - The URL of your InfluxDB host (this can your localhost, a remote instance, or InfluxDB Cloud)
+  - `QUANDL_API_KEY` - The API key fro quandl
```
Comment (jademcgough) [inline currency_exchange_rates/README.md]: "fro" should be "for"

File: currency_exchange_rates/currency_exchange_rates.yml (hunk 1 above)
Comment (russorat) [reply currency_exchange_rates/currency_exchange_rates.yml]: adding labels for the plugins being used is useful for filtering. I added more for the other plugins being used.

Comment (jsteenb2) [reply currency_exchange_rates/currency_exchange_rates.yml]: 🤔 how does one filter by label? Is that pkger specific?

Comment (russorat) [reply currency_exchange_rates/currency_exchange_rates.yml]: you can do it in the UI.

Comment (jsteenb2) [reply currency_exchange_rates/currency_exchange_rates.yml]: oh wow, had no idea this was possible 👏

Comment (wojciechka) [reply currency_exchange_rates/currency_exchange_rates.yml]: I copied that from another template so I assumed this is how I should do it.

Thank you Russ for adding the remaining labels.

**Comment by wojciechka** [comment]
> made a few slight edits, but this LGTM!

Thanks. I've added a small change to Telegraf interval (I left my testing setting of 5 seconds, whereas the data is changed daily). I also documented that fact and that you need to set longer time period to see data.

I think it's good to go, but I will wait with merging until later today in case anyone else has comments.

**Pull request merged**